trueos-handbook/introducing.rst

```diff
@@ -863,16 +965,16 @@ the screen in :numref:`Figure %s <vbox7>` displays.
    Virtual Disk - File Name and Size
 
 This screen is used to set the size (or upper limit) of the virtual
-machine. If planning to install |trueos| into the virtual machine,
-**increase the size to at least 20 GB** or an error will display during
-the |trueos| installation. If planning to install KDE, GNOME, multiple
-desktop managers, or applications within the virtual machine, choose at
-least **50 GB**. Whatever size is set, be sure the computer has enough
-free disk space to support it. Use the :guilabel:`folder` icon to browse
-to a directory on disk with sufficient space to hold the virtual
-machine.
-
-Once the selections are made, press :guilabel:`Create` to finish using
+machine. If planning to install |trueos| as the guest OS on the virtual
+machine, **increase the size to at least 20 GB** or an error will
+display during the |trueos| installation. If planning to install KDE,
+GNOME, multiple desktop managers, or applications within the virtual
+machine, choose at least **50 GB**. Whatever size is set, be sure the
+computer has enough free disk space to accommodate the size chosen. Use
+the :guilabel:`folder` icon to browse to a directory on disk with
+sufficient space to hold the virtual machine.
+
+Once the selections are made, select :guilabel:`Create` to finish using
```

Comment:
"Once the selections are made, select ..."
->
"Once the selections are made, click ..."
(Selecting selections already selected)

Reply:
👍 I'll swap click for select, but The Department of Redundancy Department may not be very happy about it :)